qhana_plugin_runner/db/models/tasks.py

```diff
@@ -83,8 +83,8 @@ class TaskData:
             )
         },
     )
-    key: str = field(metadata={"sa": Column(sql.String(100), primary_key=True)})
-    value: Optional[str] = field(metadata={"sa": Column(sql.Text(1000), nullable=True)})
+    key: str = field(metadata={"sa": Column(sql.String(500), primary_key=True)})
+    value: dict = field(metadata={"sa": Column(sql.JSON())})
```

Comment:
json can be `dict|list|str|float|int|bool|None` as every json construct (not just objects) is allowed!